examples/notion-task-github-pr-sync/index.js

```diff
@@ -0,0 +1,140 @@
+/* ================================================================================
+
+	notion-task-github-pr-sync
+  
+  Glitch example: https://glitch.com/edit/#!/notion-github-sync
+  Find the official Notion API client @ https://github.com/makenotion/notion-sdk-js/
+
+================================================================================ */
+
+function print(x) {console.log(x)}
+const { Client } = require("@notionhq/client")
+const dotenv = require("dotenv")
+const { Octokit } = require("octokit")
+const _ = require("lodash")
+
+dotenv.config()
+const octokit = new Octokit({ auth: process.env.GITHUB_KEY })
+const notion = new Client({ auth: process.env.NOTION_KEY })
+
+const databaseId = process.env.NOTION_DATABASE_ID
+const OPERATION_BATCH_SIZE = 10
+
+/**
+ * Entry Point
+ */
+updateNotionDBwithGithubPRs();
+
+/**
+ * Fetches PRs from Github and updates the according Notion Task
+ */
+async function updateNotionDBwithGithubPRs(){
+  const prs = await fetchPRsFromGitHub();
+  const prsToUpdate = []  
+  for (pr of prs){
```

Comment:
nit: there are a few instances of undeclared variables in this example, including here